.travis.yml

```diff
@@ -41,9 +43,16 @@ python:
 #  - "2.6"
   - "2.7"
 
+addons:
+  apt:
+    packages:
+      - apache2
+      - git
```

Comment:
@crepererum @jirikuncar subversion was (is) needed for the software archiver, see `invenio/utils/vcs/svn.py` and `invenio/testsuite/test_utils_vcs_svn.py` and friends.